hack/generate-proto.sh

```diff
@@ -118,7 +118,10 @@ EOF
       del(.definitions.v1alpha1OptionalArray) |
       .definitions.v1alpha1ApplicationSourcePluginParameter.properties.map = {"description":"Map is the value of a map type parameter.","type":"object","additionalProperties":{"type":"string"}} |
       del(.definitions.v1alpha1OptionalMap)
-    ' "${COMBINED_SWAGGER}" > "${SWAGGER_OUT}"
+    ' "${COMBINED_SWAGGER}" | \
+    jq '.definitions.v1Time.type = "string" | .definitions.v1Time.format = "date-time" | del(.definitions.v1Time.properties)' | \
```

Comment:
I've Wastet couple of hours researching and could not find better than do fix swagger other than manually modify it :(

Reply:
That's been my experience every time the swagger doc has been wrong. 😛